src/event_form/steps/event-form-step3.controller.js

```diff
@@ -154,6 +158,18 @@ function EventFormStep3Controller(
   };
   $scope.selectCity = controller.selectCity;
 
+  /**
+   * filter the available countries
+   * @param {boolean} isPlace
+   * @returns {Array}
+   */
+  function filterAvailableCountries(isPlace) {
+    return $scope.availableCountries.filter(function(country) {
+      // country code ZZ (bookable event) should not be visible when eventType is a place
+      return !isPlace || isPlace && country.code !== 'ZZ';
```

Comment:
If I'm not mistaken you can drop the `isPlace &&` because when you reach that part of the check it's always a place because of the `!isPlace ||`.
```suggestion
      return !isPlace || country.code !== 'ZZ';
```

Reply:
Other than this the changes look good 👍

Reply:
Works like a charm 👍